### ISSUE TITLE
Unable to login after kali update

### ISSUE THREAD

**Issue by denningsrogue** [beefproject/beef#1728]
Verify first that your issue/request has not been posted previously:

* https://github.com/beefproject/beef/issues
* https://github.com/beefproject/beef/wiki/FAQ

Ensure you're using the [latest version of BeEF](https://github.com/beefproject/beef/releases/tag/beef-0.4.7.2).

#### Environment

**Using latest version of Kali in a vm**

On what version of Ruby?

On what browser?

On what operating system?

#### Configuration

Are you using a non-default configuration? **no**

Have you enabled or disabled any BeEF extensions? **no**

#### Summary

**Following an update to kali, I am unable to login to beef.  I have confirmed the username and password are those contained in the config.yaml file.**

#### Expected Behaviour

What was the expected result?

#### Actual Behaviour

What was the actual result?

#### Steps to Reproduce

Please provide steps to reproduce this issue.

#### Additional Information

Please provide any additional information which may be useful in resolving this issue, such as debugging output and relevant screen shots. Debug output can be enabled by specifying `debug: true` in the `config.yaml` configuration file.

**Comment by denningsrogue** [comment]
I have modified config.yaml to change the credentials but I continue to get the same error for the new credentials.

The service appears to start normally, but I am unable to login.

**Comment by bcoles** [comment]
Please provide any additional information which may be useful in resolving this issue, such as debugging output and relevant screen shots. Debug output can be enabled by specifying `beef.debug: true` and `beef.http.debug` in the `config.yaml` configuration file. The debug log can be found in the `~/.beef/` directory.

**Comment by denningsrogue** [comment]
Here is the config.yaml file:  https://pastebin.com/i9X07Eri

Surprisingly, there is no ~/.beef/directory.

**Comment by bcoles** [comment]
@denningsrogue 
> 
> Here is the config.yaml file: https://pastebin.com/i9X07Eri
> 
> Surprisingly, there is no ~/.beef/ directory.

That is surprising :shrug:

It looks like your version of BeEF is out of date `version: '0.4.7.1-alpha'`.

Is your `beef-xss` package up to date? I just checked one of my Kali systems and it's using `version: '0.4.7.3-alpha'`

**Comment by denningsrogue** [comment]
Here is a screenshot of the beef service launch.
![Screen Shot 2019-07-15 at 14 02 37](https://user-images.githubusercontent.com/23585057/61245282-60acea80-a709-11e9-80f9-9fe1e77913cf.png)

And this is the login page.
![Screen Shot 2019-07-15 at 14 02 23](https://user-images.githubusercontent.com/23585057/61245302-6c001600-a709-11e9-82b0-79680b07b402.png)

**Comment by bcoles** [comment]
Your screenshot shows attempting to login with user `root`, however your config file specifies:

```yaml
    credentials:
        user:  "beef"
        passwd: "feeb"
```

**Comment by denningsrogue** [comment]
That’s because I took the screenshots at different times as I was trying different combinations.  Regardless of what username/password I specify in the config file, I get a login in error.


Richard Peters
Foothills, Alberta, Canada

587-892-4722 <tel:587-892-4722>
richardpeters@me.com <mailto:richardpeters@me.com>

**Comment by denningsrogue** [comment]
Here is the output of apt install beef-xss

apt install beef-xss
Reading package lists... Done
Building dependency tree
Reading state information... Done
beef-xss is already the newest version (0.4.7.3-0kali2).
The following packages were automatically installed and are no longer required:
  atftpd axel exe2hexbat expect gparted kali-desktop-common libgssglue1 libgtkmm-2.4-1v5 libigdgmm5 libigdgmm5:i386 php-mysql
  php7.3-mysql rdesktop snmpd tcl-expect
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.



Richard Peters
Foothills, Alberta, Canada

587-892-4722 <tel:587-892-4722>
richardpeters@me.com <mailto:richardpeters@me.com>

**Comment by denningsrogue** [comment]
I think I have solved the problem.  I did an apt purge beef-xss and then apt install beef-xss.  When I started the service it asked for a new password and now I can log in.  I suspect a failed update screwed things up.  Thanks for taking the time to respond to me.  Love your work.


Richard Peters
Foothills, Alberta, Canada

587-892-4722 <tel:587-892-4722>
richardpeters@me.com <mailto:richardpeters@me.com>
